### ISSUE TITLE
Opening a specific section in a new tab or window, or from the address bar, doesn't scroll to that section

### ISSUE THREAD

**Issue by hamirmahal** [mdn/yari#11122]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/@layer#browser_compatibility

### What information was incorrect, unhelpful, or incomplete?

Navigating to a specific section on a page, like https://developer.mozilla.org/en-US/docs/Web/API/Document/designMode#browser_compatibility, in a new tab or new private window, or from the address bar, doesn't scroll to that section.

### What did you expect to see?

Opening a section on an MDN page directly from the address bar should focus the appropriate heading, just like clicking on a section in the same page does.

**Comment by bsmth** [comment]
Thanks a lot for reporting. I've moved it over into Yari as this is the right place for platform bugs. I've also noticed this looks similar to https://github.com/mdn/yari/issues/8049 so we may close as duplicate

**Comment by mirunacurtean** [comment]
Thank you for tracking this known issue @bsmth .
Being a duplicate, I am closing this ticket, but you can foloow the discussion on https://github.com/mdn/yari/issues/8049 or https://issues.chromium.org/issues/325081538, as mentioned. Thank you for your contribution @hamirmahal .

**Comment by hamirmahal** [comment]
You're welcome @mirunacurtean.

> Thanks a lot for reporting. I've moved it over into Yari as this is the right place for platform bugs. I've also noticed this looks similar to #8049 so we may close as duplicate

You're welcome. Do you want me to make it so that there's an option to open an issue in `yari`, instead, when users try to open a new issue at https://github.com/mdn/content/issues/new/choose?

**Comment by hamirmahal** [comment]
It would look something like this.

[Screencast from 05-16-2024 03:28:41 PM.webm](https://github.com/mdn/yari/assets/43425812/3c46a266-4dcd-4722-a507-7aa7356174f2)

**Comment by mirunacurtean** [comment]
> > Thanks a lot for reporting. I've moved it over into Yari as this is the right place for platform bugs. I've also noticed this looks similar to #8049 so we may close as duplicate
> 
> You're welcome. Do you want me to make it so that there's an option to open an issue in `yari`, instead, when users try to open a new issue at https://github.com/mdn/content/issues/new/choose?

@hamirmahal No, thank you for being proactive, but this is not the case. This is the established content flow (see URL parameters; mdn/content/issues). More can be found about that and the specific repositories in the [MDN community guidelines](https://developer.mozilla.org/en-US/docs/MDN/Community/Issues) and on the [Repositories](https://developer.mozilla.org/en-US/docs/MDN/Community/Contributing/Our_repositories) page.
